### ISSUE TITLE
Connect reviewer suggestions page (wireframe 008)

### ISSUE THREAD

**Issue by g-sam** [elifesciences/elife-xpub#47]

**Comment by hdrury1** [comment]
From a user perspective, we'd like form data to be saved on clicking away from a field.

**Comment by tamlyn** [comment]
Yes, that's what we have assumed. It's not supported out of the box by the form library we are using but there is an extension for it which needs to be added. Would you like to create a ticket for it?

**Comment by tamlyn** [comment]
Oh, you already have #151
